src/test/java/org/broadinstitute/hellbender/tools/walkers/annotator/MappingQualityRankSumTestUnitTest.java

```diff
@@ -69,18 +73,64 @@ public void testMQ(){
 
         final ReferenceContext ref= null;
         final VariantContext vc= makeVC(alleleRef, alleleAlt);
-        final InfoFieldAnnotation ann = new MappingQualityRankSumTest();
 
         final Map<String, Object> annotate = ann.annotate(ref, vc, stratifiedPerReadAlleleLikelihoodMap);
 
         final double val= MannWhitneyU.runOneSidedTest(false, Arrays.asList(hardAlts[0], hardAlts[1]),
                                                               Arrays.asList(hardRefs[0], hardRefs[1])).getLeft();
         final String valStr= String.format("%.3f", val);
-        Assert.assertEquals(annotate.get(GATKVCFConstants.MAP_QUAL_RANK_SUM_KEY), valStr);
+        Assert.assertEquals(annotate.get(key), valStr);
+
+        Assert.assertEquals(ann.getDescriptions().size(), 1);
+        Assert.assertEquals(ann.getDescriptions().get(0).getID(), key);
+        Assert.assertEquals(ann.getKeyNames().size(), 1);
+        Assert.assertEquals(ann.getKeyNames().get(0), key);
+    }
+
+    @Test
+    public void testAS_MQRaw(){
+        final AS_RankSumTest ann = new AS_MappingQualityRankSumTest();
```

Comment:
I think it would be better to have separate test classes for the AS vs. non-AS annotations (eg., `MappingQualityRankSumTestUnitTest` and `AS_MappingQualityRankSumTestUnitTest` here) to make it easier to see at a glance (by looking at the file listing) which annotations are covered by tests.